javascript/ql/src/Security/CWE-022/ZipSlipBad.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
+const unzip = require('unzip');
+
+fs.createReadStream('archive.zip')
+  .pipe(unzip.Parse())
+  .on('entry', entry => {
+    const fileName = entry.path;
+    entry.pipe(fs.createWriteStream(fileName));
```

Comment:
There's normally a comment, a class name, or something in the file to indicate that this is bad.

Reply:
Done.